General_gNMI/GetLibrary.py

```diff
@@ -191,3 +191,18 @@ def get_projections_from_key_dictionary(self, key_dictionary: Dict[str, str]) ->
           """
         projections = [f"[{name}={value}]" for [name, value] in key_dictionary.items()]
         return "".join(projections)
+
+    @staticmethod
+    def count_prefix_path_steps(full_path: str):
+        """ Return number of nodes (separated with \'/\') on the specified path string. """
+        words = full_path.split('/')
```

Comment:
This is dangerous, what about paths like `/interfaces/interface{GigabitEthernet0/0/1}`? Do we control what paths will be used?

Reply:
fixed in latest codebase